File: Python/List-2/sum67.py
def sum67(nums):
  sum = 0
  record = True
  
  for i in range(len(nums)):
    if nums[i] == 6:
      record = False
    if record:
      sum += nums[i]
    if nums[i] == 7:
      record = True
      
  
  return sum
    

File: Java/Array-2/sum67.java
public int sum67(int[] nums) {
  int sum = 0;
  boolean after6 = false;
  for (int num : nums) {
    if (num == 6 && after6 == false) {
      after6 = true;
    }
    else if (num == 7 && after6 == true) {
      after6 = false;
    }
    else if (after6 == false) {
      sum += num;
    }
  }
  return sum;
}
